playground/infrastructure/ci_helper.py

```diff
@@ -47,5 +54,24 @@ def _verify_examples_status(self, examples: List[Example]):
         Args:
             examples: beam examples that should be verified
         """
-        # TODO [BEAM-13256] Implement
-        pass
+        client = GRPCClient()
+        verify_failed = False
+        for example in examples:
+            if example.status == STATUS_VALIDATION_ERROR:
+                logging.error(f"Example: {example.pipeline_id} has validation error")
+            elif example.status == STATUS_PREPARATION_ERROR:
+                logging.error(f"Example: {example.pipeline_id} has preparation error")
+            elif example.status == STATUS_ERROR:
+                logging.error(f"Example: {example.pipeline_id} has error during setup run builder")
+            elif example.status == STATUS_RUN_TIMEOUT:
+                logging.error(f"Example: {example.pipeline_id} failed because of timeout")
+            elif example.status == STATUS_COMPILE_ERROR:
+                err = await client.get_compile_output(example.pipeline_id)
+                logging.error(f"Example: {example.pipeline_id} has compilation error: {err}")
+            elif example.status == STATUS_RUN_ERROR:
+                err = await client.get_run_error(example.pipeline_id)
+                logging.error(f"Example: {example.pipeline_id} has execution error: {err}")
+            if example.status in Config.ERROR_STATUSES:
+                verify_failed = True
+        if verify_failed:
+            exit(1)  # ci step failed
```

Comment:
@KhaninArtur @ilya-kozyrev @snkalinin should we use `exit(1)` or `raise` some error?

Reply:
Sorry, I think that raising some errors will be better

Reply:
Done.